com.civicrm.hrjobroles/CRM/Hrjobroles/API/Query/ContactHrJobRolesSelect.php

```diff
@@ -129,6 +129,11 @@ private function addWhere(CRM_Utils_SQL_Select $customQuery) {
       $customQuery->where($conditions);
       unset($this->params['contact_id']);
     }
+
+    $dateRestriction = 'a.start_date <= "' . date('Y-m-d H:i:s') . '"';
```

Comment:
Better to extract this `date('Y-m-d H:i:s')` into a variable since its used twice already.
e.g `$today = date('Y-m-d H:i:s')`.

Reply:
👍 Good. Will refactor.